### ISSUE TITLE
Runtime error

### ISSUE THREAD

**Issue by kilian-gebhardt** [se4u/neural_wfst#1]
Running the training command from the README results in the following error for me: 

```
 File "<string>", line 5, in <module>
  File "$PWD/src/python/transducer_score.py", line 295, in main
    "", args, data)
  File "$PWD/src/python/lstm_seqlabel_circuit_compilation.py", line 178, in perform_training_and_testing
    return (100 - stats)
  File "/usr/lib64/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "rasengan/__init__.py", line 262, in debug_support
    sys_exc_info[1]).with_traceback(sys_exc_info[2])
AttributeError: 'exceptions.TypeError' object has no attribute 'with_traceback'
```

Commenting `.with_traceback(sys_exc_info[2])` in `rasengan/__init__.py`, line 262, will alter the last line of above traceback to:

```
TypeError: ('An update must have the same type as the original shared variable (shared_var=tparam_penalty_T_47, shared_var.type=TensorType(float32, 4D), update_val=Elemwise{mul,no_inplace}.0, update_val.type=TensorType(float64, 4D)).', 'If the difference is related to the broadcast pattern, you can call the tensor.unbroadcast(var, axis_to_unbroadcast[, ...]) function to remove broadcastable dimensions.')
```

**Comment by se4u** [comment]
Hi Kilian, I saw the issue now and I'll get back to you asap.

**Comment by se4u** [comment]
I fixed the error in the `rasengan` module. Just pull those changes by **updating rasengan**.

I think the error is happening because the `floatX` `THEANO_FLAG` is set to `float64` for some reason. Maybe its set in your `.theanorc`. The error log basically says the same thing.

```
 shared_var=tparam_penalty_T_47, shared_var.type=TensorType(float32, 4D),      
 update_val=Elemwise{mul,no_inplace}.0, update_val.type=TensorType(float64, 4D)).
```

We can fix this by adding `THEANO_FLAGS=floatX=float32` to the commandline. **I updated the command in the README**
